source/Octopus.Client/Exceptions/InvalidSpacesLimitationParametersException.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Octopus.Client.Exceptions
+{
+    public class InvalidSpacesLimitationParametersException : OctopusException
+    {
+        public InvalidSpacesLimitationParametersException(): base((int)System.Net.HttpStatusCode.Conflict, "The new spaceIds should be a subset of the previously defined spaceIds")
```

Comment:
`(int)System.Net.HttpStatusCode.Conflict`

This is a little weird since we haven't fired off a request. I don't see any reason why we should "simulate" a request exception here, so let's just not inherited from `OctopusException`.